src/main/java/tfifteenfour/clipboard/logic/commands/addCommand/AddCourseCommand.java

```diff
@@ -0,0 +1,46 @@
+package tfifteenfour.clipboard.logic.commands.addCommand;
+
+import static java.util.Objects.requireNonNull;
+
+import tfifteenfour.clipboard.logic.CurrentSelected;
+import tfifteenfour.clipboard.logic.commands.CommandResult;
+import tfifteenfour.clipboard.logic.commands.exceptions.CommandException;
+import tfifteenfour.clipboard.model.Model;
+import tfifteenfour.clipboard.model.course.Course;
+
+public class AddCourseCommand extends AddCommand {
+	public static final String COMMAND_TYPE_WORD = "course";
+	public static final String MESSAGE_USAGE = COMMAND_WORD + " " + COMMAND_TYPE_WORD
+			+ ": Adds a course. "
+            + "Parameters: "
+            + "COURSECODE\n"
+            + "Example: " + COMMAND_WORD + " " + COMMAND_TYPE_WORD
+            + " " + "CS2103T ";
```

Comment:
Some minor formatting issues
```suggestion
public class AddCourseCommand extends AddCommand {
    public static final String COMMAND_TYPE_WORD = "course";
    public static final String MESSAGE_USAGE = COMMAND_WORD + " " + COMMAND_TYPE_WORD
            + ": Adds a course. "
            + "Parameters: "
            + "COURSECODE\n"
            + "Example: " + COMMAND_WORD + " " + COMMAND_TYPE_WORD
            + " " + "CS2103T ";
```